elements/haxcms-elements/lib/core/haxcms-site-editor-ui.js

```diff
@@ -2456,7 +2456,8 @@ class HAXCMSSiteEditorUI extends HAXCMSThemeParts(
     import("@haxtheweb/video-player/lib/lecture-player.js");
     // AI chat bot
     import(
-      "@haxtheweb/haxcms-elements/lib/ui-components/magic/site-ai-chat.js"
```

Comment:
what does site-ai-chat do or can it be dropped? agent should work withoutt haxcms; site ai chat was to bridge the 2

Reply:
It was the original AI Chat interface, but everything that was in site-ai-chat was remade into chat-agent and it's subcomponents, so there's no need for it now. Deleted it in my recent commit.